src/rules/functions.rs

```diff
@@ -5,7 +5,7 @@ use crate::rules::RuleContext;
 use crate::utils::valid_group_name;
 use dino_park_trust::GroupsTrust;
 
-const NDA_GROUPS: [&str; 2] = ["nda", "contingentworkernda"];
+const NDA_GROUPS: [&str; 3] = ["nda", "contingentworkernda", "ghe_group_curators"];
```

Comment:
Please add a code comment that points us to the other place we define this. Or just put all the combined project links in all the pull requests.